### PR TITLE
Convert to TypeScript

### PR DESCRIPTION
**Pull request by avalynndev**
Requirements:
Add me to EasyScriptJS organization.
And gimme credits

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import * as modules from './modules';
+export default modules;
```
Comment (wdhdev) [inline src/index.ts:3]:
```suggestion

export default modules;
```

File: src/modules/id.ts [hunk 2]
```diff
@@ -1,21 +1,20 @@
-const { customAlphabet } = require("nanoid");
+import { customAlphabet } from "nanoid";
```
Comment (wdhdev) [inline src/modules/id.ts:1]: Will this work if I update nanoid to v4.0.0 which uses esm6?

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by wdhdev** [comment]
> Add me to EasyScriptJS organization.

I will consider it, but it is not guaranteed as this project is run by the community and it wouldnt be fair to others contributing.

**Comment by wdhdev** [comment]
> And gimme credits

You get them anyway.

File: src/modules/id.ts (hunk 2 above)
Comment (avalynndev) [reply src/modules/id.ts:1]: It should work as long as you have the 'nanoid' package installed and properly imported.

**Comment by avalynndev** [comment]
@WilliamDavidHarrison. ig i solved everything, can you merge it.
Also check the issue i created.

**Comment by avalynndev** [comment]
uhh oops. reopening..

**Review by wdhdev — APPROVED** [review]

**Pull request merged**